models/default_local_provider.go

```diff
@@ -239,3 +239,16 @@ func (l *DefaultLocalProvider) UpdateToken(http.ResponseWriter, *http.Request) {
 // TokenHandler - specific to remote auth
 func (l *DefaultLocalProvider) TokenHandler(w http.ResponseWriter, r *http.Request, fromMiddleWare bool) {
 }
+
+// ExtractToken - Returns the auth token and the provider type
+func (l *DefaultLocalProvider) ExtractToken(w http.ResponseWriter, r *http.Request) {
+	resp := map[string]interface{}{
+		"meshery-provider": l.Name(),
+		tokenName:          "",
+	}
+	logrus.Debugf("encoded response : %v", resp)
+	if err := json.NewEncoder(w).Encode(resp); err != nil {
+		logrus.Errorf("Unable to extract auth details: %v", err)
+		http.Error(w, "unable to extract auth details", http.StatusInternalServerError)
```

Comment:
I wonder whether it should be `http.StatusInternalServerError` or `http.StatusUnauthorized`

Reply:
It's opinionated though... Leave it as is if you want...

Reply:
Actually, if the user was unauthorized this function won't be called at all right. The middlewares do check if the users are authorized or not right ?

Reply:
@thakursaurabh1998, it would be good to get your eyes here.

Reply:
Agreed... But if token is invalid, I prefer it to be StatusUnauthorized... Anyways, this is a no-blocker and highly opinionated... This issue shouldn't stop this PR from getting merged.